django/shortcuts.py

```diff
@@ -43,7 +43,7 @@ def redirect(to, *args, **kwargs):
 
         * A model: the model's `get_absolute_url()` function will be called.
 
-        * A view name, possibly with arguments: `urlresolvers.reverse()` will
+        * A view name, possibly with arguments: `urls.reverse()` will
```

Comment:
could you reflow the text in cases like this to maintain line breaks near 79 characters?